ninja_extra/pagination/models/page_by_number.py

```diff
@@ -26,7 +26,7 @@
 class PageNumberPaginationExtra(PaginationBase):
     class Input(Schema):
         page: int = Field(1, gt=0)
-        page_size: int = Field(100, lt=200)
+        page_size: int = Field(100, gt=0, lt=200)
```

Comment:
Please can you change `page_size` field to `Field(100, gt=0, lt=201, le=200)`

Reply:
I don't understand why both `lt` and `le`. Isn't `n < 201` strictly equivalent to `n <= 200` when dealing with integers ?

Reply:
oh, that's a good suggestion. Please let's do that. Thanks

Reply:
I think there is a misunderstanding here. I wasn't making any suggestion, but rather pointing out that `n < 201` is strictly equivalent to `n <= 200`. This means that `Field(100, gt=0, lt=201, le=200)` is redundant. It could just be `Field(100, gt=0, lt=201)`.